base/ltfssaxes.dtx

```diff
@@ -192,503 +248,3158 @@
-\DeclareFontSeriesChangeRule {b}{sb} {sb}  {b}		%<-----
-\DeclareFontSeriesChangeRule {b}{sc}  {bsc}  {b}	%<-----
-\DeclareFontSeriesChangeRule {b}{x}  {bx}  {b}		%<-----
-%    \end{macrocode}
-%
-%    \begin{macrocode}
-\DeclareFontSeriesChangeRule {c}{bx} {bx}  {b} 		%<-----
-\DeclareFontSeriesChangeRule {c}{b}{bc}{}
-\DeclareFontSeriesChangeRule {c}{eb}{ebc}{}
-\DeclareFontSeriesChangeRule {c}{el}{elc}{}
-\DeclareFontSeriesChangeRule {c}{l}{lc}{}
-\DeclareFontSeriesChangeRule {c}{sb}{sbc}{}
-\DeclareFontSeriesChangeRule {c}{sl}{slc}{}
-\DeclareFontSeriesChangeRule {c}{ub}{ubc}{}
-\DeclareFontSeriesChangeRule {c}{ul}{ulc}{}
-\DeclareFontSeriesChangeRule {c}{x}{x}{m}               %<-----
-%    \end{macrocode}
-%
+\DeclareFontSeriesChangeRule {lc}{ul}   {ulc}  {}
+\DeclareFontSeriesChangeRule {lc}{el}   {elc}  {}
+\DeclareFontSeriesChangeRule {lc}{l}    {lc}   {}
+\DeclareFontSeriesChangeRule {lc}{sl}   {slc}  {}
+\DeclareFontSeriesChangeRule {lc}{sb}   {sbc}  {bc}
+\DeclareFontSeriesChangeRule {lc}{b}    {bc}   {}
+\DeclareFontSeriesChangeRule {lc}{eb}   {ebc}  {bc}
+\DeclareFontSeriesChangeRule {lc}{ub}   {ubc}  {bc}
+\DeclareFontSeriesChangeRule {lc}{uc}   {luc}  {}
+\DeclareFontSeriesChangeRule {lc}{ec}   {lec}  {}
+\DeclareFontSeriesChangeRule {lc}{c}    {lc}   {}
+\DeclareFontSeriesChangeRule {lc}{sc}   {lsc}  {}
+\DeclareFontSeriesChangeRule {lc}{sx}   {lsx}  {}
+\DeclareFontSeriesChangeRule {lc}{x}    {lx}   {}
+\DeclareFontSeriesChangeRule {lc}{ex}   {lex}  {}
+\DeclareFontSeriesChangeRule {lc}{ux}   {lux}  {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {lsc}{ul}  {ulsc} {}
+\DeclareFontSeriesChangeRule {lsc}{el}  {elsc} {}
+\DeclareFontSeriesChangeRule {lsc}{l}   {lsc}  {}
+\DeclareFontSeriesChangeRule {lsc}{sl}  {slsc} {}
+\DeclareFontSeriesChangeRule {lsc}{sb}  {sbsc} {bsc}
+\DeclareFontSeriesChangeRule {lsc}{b}   {bsc}  {}
+\DeclareFontSeriesChangeRule {lsc}{eb}  {ebsc} {bsc}
+\DeclareFontSeriesChangeRule {lsc}{ub}  {ubsc} {bsc}
+\DeclareFontSeriesChangeRule {lsc}{uc}  {luc}  {}
+\DeclareFontSeriesChangeRule {lsc}{ec}  {lec}  {}
+\DeclareFontSeriesChangeRule {lsc}{c}   {lc}   {}
+\DeclareFontSeriesChangeRule {lsc}{sc}  {lsc}  {}
+\DeclareFontSeriesChangeRule {lsc}{sx}  {lsx}  {}
+\DeclareFontSeriesChangeRule {lsc}{x}   {lx}   {}
+\DeclareFontSeriesChangeRule {lsc}{ex}  {lex}  {}
+\DeclareFontSeriesChangeRule {lsc}{ux}  {lux}  {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {l}{uc}    {luc}  {}
+\DeclareFontSeriesChangeRule {l}{ec}    {lec}  {}
+\DeclareFontSeriesChangeRule {l}{c}     {lc}   {}
+\DeclareFontSeriesChangeRule {l}{sc}    {lsc}  {}
+\DeclareFontSeriesChangeRule {l}{sx}    {lsx}  {}
+\DeclareFontSeriesChangeRule {l}{x}     {lx}   {}
+\DeclareFontSeriesChangeRule {l}{ex}    {lex}  {}
+\DeclareFontSeriesChangeRule {l}{ux}    {lux}  {}
+\DeclareFontSeriesChangeRule {l}{sb}    {sb}   {b}
+\DeclareFontSeriesChangeRule {l}{b}     {b}    {}
```

Comment:
```suggestion
```

This rule is superfluous. (If the third argument is equal to the second and the fourth argument is empty, then the same series is selected without the rule.)